### ISSUE TITLE
Phonetic mode not working. (Fix suggestion)

### ISSUE THREAD

**Issue by pierreverbakel** [earlephilhower/ESP8266SAM#19]
Hello Earle,

When trying out the Phonetic possibilities I noticed that SetPhonetic is not really implemented although mentioned.

I added the following lines to the file **_sam.c_** and then it appeared to work for real :

Line 17:
`
int phonetic = 0;
`

Line 61:
`
void SetPhonetic(int x) {phonetic = x;}; 
`

Phonetic Words as mentioned in the original Manual of 'Don't Ask Computer Software' do work now:
https://janderogee.com/projects/SerialSpeechSynthesisSAM/files/dict_eng.txt
e.g. 
***Q***
quality=KWAA4LIXTIY
quantity=KWAA4NTIXTIY
quart=KWOH4RT
quarts=KWOH4RTS
quebec=KUHBEH4K
question=KWEH4SCHUN
quick=KWIH4K
quiet=KWAY4IXT
quit=KWIH4T
quiz=KWIH4Z
quote=KWOW4T
quotient=KWOW4SHUNT

`void SAM_Phonetic_Test() {`
`char my_tts_text[150];`
`ESP8266SAM *sam = new ESP8266SAM;`
`sam->SetPhonetic(true); \\ Puts SAM in the Phonetic Mode  `

`strncpy_P(my_tts_text, PSTR("KWAA4LIXTIY KWAA4NTIXTIY KWOH4RT KWOH4RTS KUHBEH4K KWEH4SCHUN KWIH4K KWAY4IXT KWIH4T KWIH4Z KWOW4T KWOW4SHUNT", sizeof(my_tts_text)));`

`sam->Say(audio_output_I2S, my_tts_text);`
`sam->SetPhonetic(false);  \\ Puts SAM in the Normal Mode `

`sam->Say(audio_output_I2S, "Can you hear me now?");`
`delay(500);`
`sam->Say(audio_output_I2S, "I can't hear you!");`

` delete sam;`

`}`

PS: Thanks for the work with great library, brings back sweet memories of my early experiences on my Apple-][ Clone

**Comment by earlephilhower** [comment]
Can you make a PR with your changes?  Simply making a new variable and setting it to `true` isn't going to change the execution, so I think you probably also only called the `Parser1()` if `!phonetic`, no?

**Comment by pierreverbakel** [comment]
Hello Earle,

Reviewing my memory what happened was that the compiler stumbled over the sam->SetPhonetic(true) line.  Just before I did upgrade the libraries from your hand ESP8266Audio and ESP8266SAM. I guess the object-files that were still there after previous compiler/link runs made produced the error. After me editing the file sam.c the error went away. Probably because the old object-files were now replaced.  And then the compiler/link-error disappeared. Since the application after my changes in the sam.c file worked, I assumed that my (!) change was what made it work, and thought it to be good to report that, to the benefit of others. 
I tried to recreate the compiler-error just now, putting back older versions of the library ESP8266Audio and ESP8266SAM, AND deleting the object-files made by the compiler. But I could not. So I think my previous Issue can be closed since I believe the current state of the source-code, apparently, is ok. 

So sorry for that, but my actions and observations were based on what I saw presented on my screen. The good news: Yesterdays code was already good. 

Regards,

Pierre
